### ISSUE TITLE
mysqldump

### ISSUE THREAD

**Issue by lid3rs** [getdave/grunt-deployments#68]
mysqldump returns 'bash: mysqldump: command not found', but launching it directly on server works well.

**Comment by stuartjnelson** [comment]
Im getting the same issue. For me I wondered if it was because I'm using MAMP Pro and mySQL install is located in a different place?

**Comment by stuartjnelson** [comment]
I fixed my problem. It was what I suspected and I just needed to update my .bash-profile — Because I use MAMP Pro, mySQL is not in the standard location. So I just added this to my .bash-profile;

export PATH=/Applications/MAMP/Library/bin:$PATH

**Comment by lid3rs** [comment]
Yes, looks like it is not the programm error, it is about path of mysqldump. Case can be closed or left here for others.
